angularjs-portal-home/src/main/webapp/my-app/search/partials/search-results.html

```diff
@@ -120,7 +120,7 @@ <h4 class='header'>{{domainResultsLabel}}</h4>
 
     </div>
     <div>
-      <h4 ng-if="webSearchUrl && !googleEmptyResults"><a ng-href="{{webSearchUrl + searchText}}">View more results for {{searchText}} on {{domainResultsLabel}}</a></h4>
+      <h4 ng-if="webSearchUrl && !googleEmptyResults"><a ng-href="{{webSearchUrl + searchText}}" target=_blank>View more results for {{searchText}} on {{domainResultsLabel}}</a></h4>
```

Comment:
This would work (HTML5 makes quotes optional for attributes), but for consistency I agree, please quote.